### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Bornittah**
**Implementation**
- Develop just the UI of the calculator

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import Button from './shared/Button/Button';
+import './style.css';
+
+const Calculator = () => (
```
Comment (mikethreels) [inline src/components/Calculator.jsx]: - [ ] the `Calculator` component should be class-based according to the requirements

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <Calculator />
+  );
+}
```
Comment (mikethreels) [inline src/App.jsx]: - [ ] the `App` component should be class-based according to the requirements

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Bornittah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] in the public and src folder there are still some files which are created by the CRA which need to be removed.
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 

Hi @Bornittah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**